test/jdk/java/nio/channels/SocketChannel/CloseRegisteredChannel.java

```diff
@@ -42,8 +42,8 @@ public static void main(String[] args) throws Exception {
 
         SocketChannel client = SocketChannel.open ();
         client.connect (new InetSocketAddress (InetAddress.getLoopbackAddress(), port));
-        SocketChannel slave = server.accept ();
-        slave.configureBlocking (true);
+        SocketChannel channel = server.accept ();
```

Comment:
Can you rename this to "peer" instead? (we can remove the spurious space after accept while we are there).

Reply:
Yes, I will name it, "peer", and remove the extra space on the affected lines.